lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -99,7 +99,7 @@ public static function unregister_font_collection( $slug ) {
 			_doing_it_wrong(
 				__METHOD__,
 				/* translators: %s: Font collection slug. */
-				sprintf( __( 'Font collection "%s" not found.', 'default' ), $slug ),
```

Comment:
Thanks for catching these in this file! These are not (yet) existing Core strings, so the text domain should be 'gutenberg'.